tools/CACertificates/certGen.sh

```diff
@@ -367,7 +367,7 @@ function generate_verification_certificate()
 ###############################################################################
 # Generates a certificate for a device, chained to the intermediate.
 ###############################################################################
-function generate_device_certificate()
+function generate_device_identity_certificate()
```

Comment:
why certGen.sh doesn't show up in 1.0.9 release PR, https://github.com/Azure/iotedge/pull/2022/files

Reply:
I was going to ask you about this tomorrow. I forgot to add in 1.0.9, however everything worked fine. I have added here so as to prevent future issues due to old files. Should I open a separate PR to port to 1.0.9?

Reply:
Will open new PR for 1.0.9